ArduPlane/quadplane.cpp

```diff
@@ -2932,6 +2937,9 @@ void QuadPlane::vtol_position_controller(void)
     case QPOS_LAND_DESCEND:
     case QPOS_LAND_ABORT:
     case QPOS_LAND_FINAL: {
+        if (tailsitter.enabled() && now_ms-last_pos2_ms<7000) {
```

Comment:
Really not a fan of adding a hard coded delay. Why seven seconds? If you don't want to start landing why not delay the switch into the land phase rather than switching to land and then not landing?

Reply:
Where in the code?

Reply:
For example, if the tailsitter performs the transition perfect over the home point with the weathervane active this allows him to perform the yaw rotation before descending, which is important.